deployment/docker-compose.yml

```diff
@@ -61,11 +60,23 @@ uwsgi:
     - ./media:/home/web/media:rw
     - ./reports:/home/web/reports
     - ./logs:/var/log/
+  user: root
+
+uwsgi-ephemeral:
+  extends: uwsgi-base
+  container_name: mezzanine-uwsgi-ephemeral
```

Comment:
@cchristelis If there is no container name, how do i do to migrate/shell to this container?